packages/cli/src/lib/Web3API.ts

```diff
@@ -12,10 +12,8 @@ export class Web3API {
     if (!manifest) {
       throw Error(`Unable to parse manifest: ${manifestPath}`);
     }
-
-    // TODO: add validation
-    // - validate manifest structure
-    // - ensure everything exists
+  
+    manifestValidation(manifest)
```

Comment:
I think that here, we should redefine the `manifest` variable. The reason is that on this function, first, we check if the manifest passed as argument is outdated, if that's true, then it's going to be updated, hence, editing this variable that is being passed. If we don't update the `manifest` variable here, it will compile the older version of the manifest.

Reply:
Signature should be something like this: `manifestValidation(manifest: AnyManifest): LatestManifest`, so we can set `manifest` equal to return value.

Also change the function name to `sanitizeAndUpgrade`

Reply:
Also change the import path from root level to some nested folder like `@web3api/client-js/manifest`, this way top level imports is very minimal and easy to understand.